cypress/e2e/hub/collections.cy.ts

```diff
@@ -2,16 +2,19 @@ import { randomString } from '../../../framework/utils/random-string';
 import { hubAPI } from '../../support/formatApiPathForHub';
 import { Collections } from './constants';
 
-describe('Collections- List View', () => {
+describe.skip('Collections- List View', () => {
```

Comment:
@acruzgon is that the intent to skip those tests?

Reply:
yes, temporarily skipping until they are less flaky @nixocio